ios/xcode/MGLKit/MGLLayer.h

```diff
@@ -23,6 +23,7 @@ typedef enum MGLDrawableColorFormat : int
     MGLDrawableColorFormatRGBA8888  = 32,
     MGLDrawableColorFormatSRGBA8888 = -32,
     MGLDrawableColorFormatRGB565    = 16,
+    MGLDrawableColorFormatRGBA16    = 64,
```

Comment:
Currently BT.2020 colorspace is only used with RGBA16. Maybe instead of creating a separate `MGLDrawableColorSpace` property, you would create 2 separate color formats like this:
- Linear: `MGLDrawableColorFormatRGBA16` (I don't think this format is handled properly inside SurfaceMtl without BT.2020 colorspace as of now)
- BT.2020: `MGLDrawableColorFormatRGBA16BT2020PQ` (this will propagate 16 bits channels config and BT.2020 color space to EGL)

This is the same as sRGB currently can only be used with RGBA8 format, hence it has a dedicated entry `MGLDrawableColorFormatSRGBA8888 `.

Reply:
Great, I very much like this idea instead of my current implementation. I will switch things over.